app/views/ExposureHistory/ExposureDatumDetail.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import dayjs from 'dayjs';
+
+import { Typography } from '../../components/Typography';
+
+import {
+  Typography as TypographyStyles,
+  Outlines,
+  Colors,
+  Spacing,
+} from '../../styles';
+
+import {
+  ExposureDatum,
+  Possible,
+  Unknown,
+  NoKnown,
+} from '../../ExposureNotificationContext';
+
+interface ExposureDatumDetailsProps {
+  exposureDatum: ExposureDatum;
+}
+
+const ExposureDatumDetail = ({
+  exposureDatum,
+}: ExposureDatumDetailsProps): JSX.Element => {
+  switch (exposureDatum.kind) {
+    case 'Possible': {
+      return <PossibleExposureDetail datum={exposureDatum} />;
+    }
+    case 'Unknown': {
+      return <UnknownExposureDetail datum={exposureDatum} />;
+    }
+    case 'NoKnown': {
+      return <NoKnownExposureDetail datum={exposureDatum} />;
+    }
+  }
+};
+
+interface PossibleExposureDetailProps {
+  datum: Possible;
+}
+
+const PossibleExposureDetail = ({
```

Comment:
What are your thoughts about having these all in the same file? Are they three different components?

Reply:
They are all quite similar at this point and functionally service the same purpose, so i left them in. But I could see a world where they each have their own file. I see it as style preference at this point.